sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -69,6 +71,11 @@
 except ImportError:
   pass
 
+try:
```

Comment:
I had to explicitly import this otherwise many tests failed with "apache_beam.io.gcp.internal.clients.bigquery has no attribute TableReference" - I guess bigquery lib was not available in some environments. The pylint check also failed possibly because of this but I haven't figured out what's wrong. Any insights?